core/state_transition.go

```diff
@@ -613,9 +613,12 @@ func (st *stateTransition) innerExecute() (*ExecutionResult, error) {
 		}, nil
```

Comment:
We should check if we need to set the `MaxUsedGas` field pre-Regolith. The `RefundedGas` field got removed and a new `MaxUsedGas` field was added.

Reply:
I've checked and the only place where the `MaxUsedGas` from the `ExecutionResult` is used is in the gas estimator at `eth/gasestimator/gasestimator.go` - and this is irrelevant to deposit txs. So I think it's safe to keep not setting it.